src/main/java/frc/robot/Robot.java

```diff
@@ -123,4 +130,8 @@ public void testInit() {
 	@Override
 	public void testPeriodic() {
 	}
+
+	public static CVData getCVData() {
```

Comment:
This is not great, we should treat Jetson kind of like a subsystem in terms of scoping. In autonomousInit, we assume we have a value already after calling jetson.updateVision(), so no reason the path can't be passed in as a parameter to the constructor of CVDriveCommand. This might require more refactoring that is feasible by tonight though.